__tests__/index.tests.js

```diff
@@ -39,3 +40,63 @@ describe('error 404', ()=>{
         })
     })
    
+    describe('GET /api', () => {
+      test('GET:200 sends an object with the endpoints to the client', () => {
+        return request(app)
+          .get('/api')
+          .expect(200)
+          .then((response) => {
+            expect(response.body.endpoints).toEqual(expect.any(Object));
```

Comment:
Please remember that both objects and arrays have a data type of object, so this would pass in both cases